src/cdcacm_serial.c

```diff
@@ -171,7 +171,7 @@ void init_cdc_acm()
 	gpio_pin_write(gpio_dev, TXRX_LED, 1);
 }
 
-void cdcacm_setup(void)
+void cdcacm_setup(void *dummy1, void *dummy2, void *dummy3)
```

Comment:
So no warnings anymore, then? If there are no compile warnings I'm happy with not using ARG_UNUSED